redfish-core/lib/hypervisor_system.hpp

```diff
@@ -1237,13 +1258,15 @@ inline void requestRoutesHypervisorSystems(App& app)
         std::optional<std::string> ipv6OperatingMode;
         std::optional<nlohmann::json> statelessAddressAutoConfig;
         std::optional<bool> ipv6AutoConfigEnabled;
+        std::optional<std::string> ipv6DefaultGateway;
 
         if (!json_util::readJsonPatch(
                 req, asyncResp->res, "HostName", hostName,
                 "IPv4StaticAddresses", ipv4StaticAddresses,
                 "IPv6StaticAddresses", ipv6StaticAddresses, "IPv4Addresses",
                 ipv4Addresses, "DHCPv4", dhcpv4, "DHCPv6", dhcpv6,
-                "StatelessAddressAutoConfig", statelessAddressAutoConfig))
+                "StatelessAddressAutoConfig", statelessAddressAutoConfig,
+                "IPv6DefaultGateway", ipv6DefaultGateway))
```

Comment:
This is readonly property. Can not patch.

For patching you must use IPv6StaticDefaultGateways property

Reply:
Done